akamai_appsec.json.example

```diff
@@ -0,0 +1,5 @@
+[
+{"status":"covered","hasMatchTarget":true,"hostname":"hola.example.com.mx","policyNames":["MainStreetCorporationMexico"],"configuration":{"id":57415,"name":"MainStreetCorporationMexico","version":37}},

```

Comment:
Can we pretty print these JSON blobs here

Reply:
The previous examples were not pretty printed, so I left things as-is. I could add this as a feature, though, no problem

Reply:
@ccloes would that cause issues? was just throwing it out there as it was the only thing that stood out to me looking through the PR

Reply:
I am ok either way.  But we talked about that today, the first thing I do is format the docs when viewing them.  There won't be any issue with the way they look as long as they are still JSON format.